polaris-react/src/components/Button/Button.scss

```diff
@@ -874,6 +876,13 @@
   &.sizeLarge {
     padding-left: var(--p-space-3);
     padding-right: var(--p-space-3);
+
+    #{$se23} & {
+      // stylelint-disable -- polaris/conventions/polaris/custom-property-allowed-list
+      padding-left: var(--pc-button-large-vertical-padding);
+      padding-right: var(--pc-button-large-vertical-padding);
```

Comment:
Should we use `var(--p-space-1_5-experimental)` directly? I'm assuming that will be the default after cleaning up this component.